spawn_sdk/spawn_sdk/README.md

```diff
@@ -82,17 +82,19 @@ defmodule SpawnSdkExample.Actors.MyActor do
 
   alias Io.Eigr.Spawn.Example.{MyState, MyBusinessMessage}
 
-  defact init(%Context{state: state} = ctx) do
+  # The callback could also be referenced to an existing function:
+  # action "init", &some_defp_handler/0
```

Comment:
I think the init action is a special case that should be worth being more explicit. Maybe via macro init instead of macro action.